static/js/lib/selectors.js

```diff
@@ -0,0 +1,7 @@
+// @flow
```

Comment:
`selectors` is a bit ambiguous. Maybe `state_selectors`, `state_helpers`, or `state_utils` would be more helpful?

Reply:
sure, changing to `redux_selectors`